packages/elements-react/README.md

```diff
@@ -93,56 +93,101 @@ export function init(params: QueryParams, flowType: FlowType) {
 
 To access & render the flow, on your flow page, you can use the `flow` query
 parameter, that is included in the redirect. Use it to call the
-[`getLoginFlow()`](https://www.ory.sh/docs/reference/api#tag/frontend/operation/getLoginFlow)
+[`getRegistrationFlowRaw()`](https://www.ory.sh/docs/reference/api#tag/frontend/operation/getRegistrationFlow)
 API.
 
 **Full Example**:
 
-````ts
-export async function getOrCreateRegistrationFlow(
-  params: QueryParams
-): Promise<RegistrationFlow> {
-  const onRestartFlow = () => init(params, FlowType.Registration);
+```ts
+import {
```

Comment:
Do we need these examples? We could probably just embed the source code from our react pages / react app router example?